free-2.md

```diff
@@ -58,12 +135,34 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Since these criteria are most easily described in natural language, it would
+be a significant challenge translating it into code.
+So a lot of time would need to be dedicated to syntax and naming.
+
+However, semantics would also be difficult as the program would need to
+interact with an email client in a secure way.
+Another challenge in semantics would be natural language processing problems 
+that may be required in order to filter things such as counting threads.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Due to the challenges in communicating with an email client and my unfamiliarity
+with NLP, this would be an ambitious project for me. The problem is quite open-
+ended so it would also be a challenge to determine how much the DSL should be
+able to handle.
```

Comment:
I also imagine the whole filtering part could be tricky since you would have to know what emails arrive before the user does. Also to work with email accounts of many types sounds real hard to me lol.